ibm/mas_devops/roles/kubeturbo/templates/kubeturbo-cr.yaml

```diff
@@ -0,0 +1,17 @@
+---
+# More options:
+apiVersion: charts.helm.k8s.io/v1
+kind: Kubeturbo
+metadata:
+  name: "{{ kubeturbo_cr_name }}"
+  namespace: "{{ kubeturbo_namespace }}"
+spec:
+  restAPIConfig:
+    opsManagerPassword: "{{ turbonomic_password }}"
+    opsManagerUserName: "{{ turbonomic_username }}"
```

Comment:
Rather than set these two values in the CR we should be setting restAPIConfig.turbonomicCredentialsSecretName and pointing to a secret name that contains these.

Reply:
I am not familiar with this process and need to do some investigation to implenment the change.

Reply:
@pseemung it should just be setting it like shown here https://github.com/search?q=repo%3Aturbonomic%2Fkubeturbo%20turbonomicCredentialsSecretName&type=code and having an update in the ansible-devops to create that secret as well. What was the error in the kubeturbo operator or pod?